src/backend/expungeservice/setup.py

```diff
@@ -6,9 +6,10 @@
     version='0.1',
     description='recordexpungPDX Expunge Service',
     packages=['crawler',
+              'database',
               'endpoints',
               'expunger',
-              'test'],
+              'models'],
```

Comment:
Is this something we can do dynamically instead of hardcoding the values in?

Reply:
I'm not familiar enough with packaging to know if there is a conventional dynamic solution, but I'm happy to look into it. Meanwhile I'd like to just merge this for now.